modules/local/fastptrim/main.nf

```diff
@@ -0,0 +1,41 @@
+/* 
+Trim poor quality reads from fastq input files using Fastp.
+*/
+
+process FASTP_TRIM {
+    tag "$meta.id"
+    label 'process_single'
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/fastp:0.23.4--h5f740d0_0' :
+        'biocontainers/fastp:0.23.4--h5f740d0_0' }"
+
+    input:
+    tuple val(meta), path(reads)
+
+    output:
+    tuple val(meta), path("*_trimmed.fastq.gz"),    emit: reads
+    tuple val(meta), path("*.json"),                emit: json
+    tuple val(meta), path("*.html"),                emit: html
+    path "versions.yml",                            emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args = task.ext.args ?: ''
+    def prefix = task.ext.prefix ?: "${meta.id}"
+    
+    if(meta.single_end || reads instanceof nextflow.processor.TaskPath) {
+        args = args + "-i ${reads[0]} -o ${meta.id}_trimmed.fastq.gz"
+    }else{
+        args = args + "-i ${reads[0]} -I ${reads[1]} -o ${meta.id}R1_trimmed.fastq.gz -O ${meta.id}R2_trimmed.fastq.gz"
```

Comment:
Could you add an underscore to the output file name. As in: `${meta.id}_R1_trimmed.fastq.gz`?

Reply:
No problem - updated in [7f8eb23](https://github.com/phac-nml/speciesabundance/pull/3/commits/7f8eb237ed497bc687246657943c1c05b1da0a00)

Reply:
Thanks so much 😄